operator/cmd/mesh/manifest-apply.go

```diff
@@ -160,7 +169,22 @@ func ApplyManifests(setOverlay []string, inFilenames []string, force bool, dryRu
 		l.logAndPrint("\n\n✘ Errors were logged during apply operation. Please check component installation logs above.\n")
 		return fmt.Errorf("errors were logged during apply operation")
 	}
-
 	l.logAndPrint("\n\n✔ Installation complete\n")
+
+	if err := saveClusterState(iops, installedSpecCRPrefix+iops.Revision, opts); err != nil {
```

Comment:
if revision is not set this will result in the name "installed-state-" I think (I am surprised this is even allowed). Maybe should just drop the `-` if its not set, or make it `installed-state-default`?

Reply:
done